src/components/Explorer/Base/Result/RunLookup/RunLookup.tsx

```diff
@@ -62,8 +85,14 @@ export const RunLookup = ({ runId, filters }: Props) => {
                     />
                 </div>
             </div>
+
             <hr className='mb-4' />
             {!sequenceResult ? instructions : sequenceResult}
+            <hr className='m-4' />
```

Comment:
```suggestion
            <hr className='my-4' />
```

`m-4` adds a margin to left and right, making it look like a "subsection" of the above section when it is actually unrelated.

<img width="600" alt="m-4" src="https://user-images.githubusercontent.com/13424970/156326861-7ca2572b-607c-4d19-ad97-7be00443672f.png">

`my-4` keeps the space above/below the line without shrinking width.